source/Reference.js

```diff
@@ -1,9 +1,9 @@
-module.exports.GAME_WIDTH = 230
-module.exports.GAME_HEIGHT = 130
+module.exports.GAME_WIDTH = 480
+module.exports.GAME_HEIGHT = 270
```

Comment:
I think this might be a bit _too_ big. Satisfying play area, but the Sprites feel a bit dinky. What do you think about dragging it down to 400 X 225 or so?